docs/examples/mission_basic.rst

```diff
@@ -0,0 +1,168 @@
+.. _example_mission_basic:
+
+======================
+Example: Basic Mission
+======================
+
+This example demonstrates the basic mission operations provided by DroneKit-Python, including:
+downloading missions from the vehicle, clearing missions, creating mission commands
+and uploading them to the vehicle, monitoring the current active command, and changing the active 
+command. 
+
+The guide topic :ref:`auto_mode_vehicle_control` provides more detailed explanation of how the API
```

Comment:
could also mention "how the API should be used, and in particular the operation of AUTO mode."